### PR TITLE
UIs: rewrite components by using Hooks

### LINKED ISSUE #1943: UI: Replace react-redux's connect() by Hooks
**Component**: UI

**Why this is needed**:
- As we begin to use React Hooks, It would be nice to use react-redux's Hooks as well to be consistent.
See: https://react-redux.js.org/next/api/hooks
- Same thing for react-router
See: https://reacttraining.com/react-router/core/api/Hooks

**What should be done**:

**Implementation proposal** (strongly recommended):

**Test plan**:

### PR DESCRIPTION
**Pull request by carlito-scality**
**Component**:UIs

**Context**: 
- As we begin to use React Hooks, It would be nice to use react-redux's Hooks as well to be consistent.
See: https://react-redux.js.org/next/api/hooks
- Same thing for react-router
See: https://reacttraining.com/react-router/core/api/Hooks

**Summary**:

**Acceptance criteria**: 

---

Closes: https://github.com/scality/metalk8s/issues/1943

### CODE REVIEW

File: ui/src/containers/CreateVolume.js [hunk 1]
```diff
@@ -88,16 +89,22 @@ const SizeUnitFieldSelectContainer = styled.div`
...
+  const storageClass = useSelector(state => state.app.volumes.storageClass);
+  const theme = useSelector(state => state.config.theme);
 
   useEffect(() => {
-    fetchStorageClass();
-  }, [fetchStorageClass]);
+    dispatch(fetchStorageClassAction());
+  }, [dispatch]);
 
   const nodeName = props.match.params.id;
```
Comment (ChengYanJin) [inline ui/src/containers/CreateVolume.js]: I am wondering can we use useParams() hook to get the params from particular route?

File: ui/src/containers/CreateVolume.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React, { useEffect } from 'react';
-import { connect, useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { useRouteMatch, useHistory } from 'react-router';
 import { Formik, Form } from 'formik';
 import * as yup from 'yup';
```
Comment (ChengYanJin) [inline ui/src/containers/CreateVolume.js:5]: I found that some components we use `import * as Yup from 'yup'`. It's better we can keep them consistent. (Maybe not very relevant to this issue)

**Review by ChengYanJin — COMMENTED** [review]
One minor spelling mistake:
The title of some commit messages write`componeny` instead of `component`.

**Review by ChengYanJin — APPROVED** [review]
LGTM!

**Comment by carlito-scality** [comment]
@bert-e approve

**Pull request merged**